azkaban-common/src/main/java/azkaban/Constants.java

```diff
@@ -37,6 +37,9 @@
 
   public static final String AZKABAN_SERVLET_CONTEXT_KEY = "azkaban_app";
 
+  // Path name of execute-as-user executable
+  public static final String NATIVE_LIB_FOLDER = "azkaban.native.lib";
```

Comment:
There is a configuration key with this name already. Let's have this in `Constants.ConfigurationKeys`